pkg/module/resources.go

```diff
@@ -45,23 +40,8 @@ func AddResources(
 	fs vfs.FileSystem,
 	registryCredSelector string,
 ) error {
-	descriptor := archive.GetDescriptor()
-	var matchLabels []byte
-	if registryCredSelector != "" {
-		selector, err := metav1.ParseToLabelSelector(registryCredSelector)
-		if err != nil {
-			return err
-		}
-		matchLabels, err = json.Marshal(selector.MatchLabels)
-		if err != nil {
-			return err
-		}
-		descriptor.SetLabels([]ocmv1.Label{{
```

Comment:
Why is this removed?

Reply:
Labels are added here: https://github.com/kyma-project/cli/blob/4bbba5934c0e77cbb38b6a7162eb4c182bf40a98/pkg/module/resources.go#L69